sdk/eventhub/azure-eventhubs/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -0,0 +1,187 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import logging
+from typing import Any, Union, TYPE_CHECKING, Callable, Dict, List, Tuple
+from azure.eventhub import EventPosition, EventData, \
```

Comment:
Which version of [black](https://pypi.org/project/black/) formatted the code like this? All versions I've used will break it up as:

```python
from x import (
    one,
    two
)
```

Reply:
Changed to one line. I didn't use black to format this file.

Reply:
Not blocking for preview5, solving